packages/bdd-tests/step_definitions/shared-steps.ts

```diff
@@ -0,0 +1,113 @@
+import { getToken } from "../services/getaccessToken";
```

Comment:
It would make more sense to define these inside the other files in this folder. authentication is the only one that I would say is actually shared.

Reply:
You can't have the whole function inside the shared steps. It is not clean to have the get access Token steps in the shared step. Shared steps is ideal for putting the getToken call, which I am doing. But the whole logic should be put in a different file. May I am missing your point here